### ISSUE TITLE
PmenuSel color is not right in my setup

### ISSUE THREAD

**Issue by jrop** [tomasiser/vim-code-dark#47]
I hope this is not spam, but I figured that you would be able to point me in the right direction to debug this.  When I open a fresh instance of NeoVim, this is the setting for PmenuSel in a tsx file:

<img width="588" alt="Screen Shot 2020-02-20 at 3 10 36 PM" src="https://user-images.githubusercontent.com/1404053/74983653-52d3ec00-53f3-11ea-845e-ac9640585f71.png">

However, the color does not show up right in a pop-up menu (Coc):

<img width="934" alt="Screen Shot 2020-02-20 at 3 09 33 PM" src="https://user-images.githubusercontent.com/1404053/74984282-739c4180-53f3-11ea-9034-e6e6e4d2c43e.png">

After clearing the color (`:hi clear PmenuSel`), the highlight shows up, albeit dark:

<img width="932" alt="Screen Shot 2020-02-20 at 3 10 00 PM" src="https://user-images.githubusercontent.com/1404053/74984846-93cc0080-53f3-11ea-8dd6-cc933a420478.png">

Basically, I'm not sure why the teal color is not showing up when the menu is selected, and I am not intimately familiar with how the highlighting scheme works in Vim (although I am attempting to read through some docs on it).

**Comment by jrop** [comment]
Oops, for anybody else reading this: the issue had to do with terminal colors: I had to add the following lines to my `.tmux.conf`:

```
set -ga terminal-overrides ",xterm-256color:RGB"
set -g default-terminal "tmux-256color"
```

I narrowed the problem down to being TMUX-related when I thought to test the colors outside of TMUX (they worked just fine).

Sorry for the noise, and thanks for an incredibly soothing theme!

**Comment by tomasiser** [comment]
Hi, glad you solved the issue! :-)
